apps/meteor/ee/app/livechat-enterprise/server/hooks/afterForwardChatToDepartment.js

```diff
@@ -13,7 +15,7 @@ callbacks.add(
 			cbLogger.debug('Skipping callback. No room found');
 			return options;
 		}
-		LivechatRooms.unsetPredictedVisitorAbandonmentByRoomId(room._id);
+		LegacyLivechatRoomsModel.unsetPredictedVisitorAbandonmentByRoomId(room._id);
```

Comment:
Can you migrate this function to the raw model?

Reply:
Done :smile: